app/add-recipe.html

```diff
@@ -65,11 +65,7 @@ <h2 class="title-new">Add New Recipe</h2>
     </form>
     <footer>
       <section class="footer-features">
-        <form class="" action="index.html" method="post">
-          <label class="search-label" for="search">Hungry?</label>
-          <input type="text" class="search-input" name="search" placeholder="Search for recipe" />
-          <input type="submit" class="footer-button search-button" value="Search" disabled />
-        </form>
+        <label class="search-label" for="search">Hungry?</label>
```

Comment:
We can probably just change to something not a label since there's no input field